src/main/java/com/example/bookstore/validator/FieldMatchValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.example.bookstore.validator;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.beanutils.BeanUtils;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String firstFieldName;
```

Comment:
names like first and second do not contain any context, for me it is not clear what first and second  means, please choose more suitable names for this fields